### DISCUSSION TITLE
插件不能调用注册中心的服务

### DISCUSSION THREAD

**Discussion by dengfenghong** [apache/apisix#7845] [Q&A]
### Description

插件不能调用注册中心的服务，在一些内部鉴权的接口，不能直接暴露在网关路由里面，需要走注册中心或者直接调用

**Comment by tzssangglass** [comment]
Please describe the problem clearly and give a minimalistic reproduction of the use case

**Reply by dengfenghong** [reply]
重写JWT插件后，JWT的token生成、检查用的是自写的接口，如果要从插件中调用这个自写的接口，需要把接口配置到路由列表中，但是这样会导致自写的接口暴露给外网。我们想采用JWT插件消费nacos、zookeeper等注册中心提供的接口。但是目前没有发现插件对接注册中心调用服务的案例

**Reply by tzssangglass** [reply]
you can implate it by yourself
